packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Resiliency/SrvDriven/SrvDrivenTests.V1.cs

```diff
@@ -39,7 +39,7 @@ public Task AddOptionalParamFromOneOptional_V1Client_V1Service_WithApiVersionV1(
         {
             var options = new ResiliencyServiceDrivenClientOptions(ResiliencyServiceDrivenClientOptions.ServiceVersion.V1);
             var client = new ResiliencyServiceDrivenClient(host, ServiceDeploymentV1, options);
-            var response = await client.FromOneOptionalAsync("optional");
+            var response = await client.FromOneOptionalAsync("optional", cancellationToken: default);
```

Comment:
Why did we need to do this?

Reply:
After regen, this call was ambiguous between the convenience method and the protocol method. Looking at this closer now, the protocol method has the optional query param as being required and the RequestOptions param as being optional. Looks like we are explicitly removing the default value from the optional param in the protocol method and making the request options param optional. Do you know if this is expected?

Reply:
Ah I see. Yeah this sounds like a bug. If the convenience method and the protocol method have the same parameters other than ct/request options, then request options needs to be required regardless of the requiredness of the earlier parameters.

Reply:
Thanks, I will push a fix for this then.